frontend/shared/browser-tests/utils/testcafe.utils.ts

```diff
@@ -15,24 +15,23 @@ const injectCallback = (
   obj: typeof screen | typeof within,
   callback: (...args: unknown[]) => void
 ): typeof obj =>
-  Object.keys(obj)
-    // eslint-disable-next-line @typescript-eslint/ban-ts-comment
-    // @ts-ignore
-    // eslint-disable-next-line @typescript-eslint/no-unsafe-member-access
-    .filter((key: Key) => typeof obj[key] === 'function')
-    .reduce(
-      (acc, key) => ({
-        ...acc,
-        [key]: (...params: unknown[]) => {
+  Object.fromEntries(
```

Comment:
eslint autofixed the usage of object.keys + reduce to object.fromEntries + map which seems to be great autofix